ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java

```diff
@@ -273,7 +278,8 @@ private static HttpServerOptions createHttpServerOptions(final ApiServerConfig a
           throw new ConfigException("Invalid URI scheme should be http or https: " + listenerName);
         }
         if ("https".equalsIgnoreCase(scheme)) {
-          final String keyStoreLocation = config.getString(ApiServerConfig.TLS_KEY_STORE_PATH);
+          final String keyStoreLocation = config
+              .getString(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG);
```

Comment:
nit: can we use `KsqlRestConfig.SSL_KEYSTORE_LOCATION_CONFIG` instead? And similarly for the other appearances of `SslConfigs` in this file, and in ListenersTest.java and TlsTest.java.

Reply:
ack

Reply:
We don't redefine these constants in KsqlRestConfig - as there doesn't seem to be any point - and we want them to be the same anyway.